systemvm/patches/debian/buildsystemvm.sh

```diff
@@ -375,8 +375,10 @@ packages() {
   chroot . apt-get --no-install-recommends -q -y --force-yes install open-vm-tools
   #xenstore utils
   chroot . apt-get --no-install-recommends -q -y --force-yes install xenstore-utils libxenstore3.0
-  #keepalived and conntrackd
-  chroot . apt-get --no-install-recommends -q -y --force-yes install keepalived conntrackd ipvsadm libnetfilter-conntrack3 libnl1
+  #keepalived - install version 1.2.13 from wheezy backports
+  chroot . apt-get --no-install-recommends -q -y --force-yes -t wheezy-backports install keepalived
```

Comment:
Any package related change needs to go into:  tools/appliance/definitions/systemvmtemplate/install_systemvm_packages.sh  
AFAIK, this buildsystemvm script is not used or maintained by anyone